package.json

```
@@ -8,10 +8,10 @@
     "url": "git+https://github.com/debitoor/schemagic.git"
   },
   "private": false,
-  "main": "./source/schemagic.js",
+  "main": "./lib/index.js",
   "engines": {
-    "node": ">=0.10.28",
-    "npm": ">=2.4.1"
+    "node": ">=16.20",
+    "npm": ">=8.19"
```

Might use use major versions:

```suggestion
    "node": ">=16",
    "npm": ">=8"
```